docs/nitro-deploy.md

```
@@ -1328,6 +1328,109 @@ sudo systemctl restart vsock-tinfoil-router-inf9.service
+```
+- {address: cloud-api.near.ai, port: 443}
+- {address: nras.attestation.nvidia.com, port: 443}
+- {address: api.trustedservices.intel.com, port: 443}
+```
+
+Restart the nitro vsock proxy service:
+```
+sudo systemctl restart nitro-enclaves-vsock-proxy.service
+```
+
+#### Near.AI Cloud API
```

markdownlint-cli2 (0.21.0)

[warning] Heading levels should only increment by one level at a time
Expected: h3; Actual: h4

(MD001, heading-increment)